tests/Testcontainers.InfluxDb.Tests/InfluxDbContainerTests.cs

```diff
@@ -0,0 +1,31 @@
+namespace Testcontainers.InfluxDb;
+
+public sealed class InfluxDbContainerTest : IAsyncLifetime
+{
+    private readonly InfluxDbContainer _influxDbContainer = new InfluxDbBuilder().Build();
+
+    public Task InitializeAsync()
+    {
+        return _influxDbContainer.StartAsync();
+    }
+
+    public Task DisposeAsync()
+    {
+        return _influxDbContainer.DisposeAsync().AsTask();
+    }
+
+    [Fact]
+    [Trait(nameof(DockerCli.DockerPlatform), nameof(DockerCli.DockerPlatform.Linux))]
+    public async Task PingReturnsTrue()
+    {
+        // Given
+        using var client = new InfluxDBClient(_influxDbContainer.GetAddress());
+
+        // When
+        var result = await client.PingAsync()
+            .ConfigureAwait(false);
+
+        // Then
+        Assert.True(result);
```

Comment:
Can we add a slightly more complex test that requires authentication (the tests are usually a good start or example for other developers )?

Reply:
Done ;)

Reply:
Thanks